CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Updated swagger example values (#139)
 - Extract out service logic from StreamDao to a new Service Layer (#140)
+- Fixed Exception Handling to catch Runtime Exception in Cluster Resource
+- Fixed the response type to text in Stream Resource
 
```

Comment:
i believe the numbers at the end are PR numbers.
so can we add (#141) at the end.

Reply:
Yes!